cmd/bitrot-streaming.go

```diff
@@ -56,9 +56,15 @@ func (b *streamingBitrotWriter) Write(p []byte) (int, error) {
 	hashBytes := b.h.Sum(nil)
 	_, err := b.iow.Write(hashBytes)
 	if err != nil {
+		b.closeWithErr(err)
 		return 0, err
 	}
-	return b.iow.Write(p)
+	n, err := b.iow.Write(p)
+	if err != nil {
+		b.closeWithErr(err)
+		return 0, err
```

Comment:
I think the best is to return 'n, err' here, you don't know, maybe an error occurs in the middle of writing